frontend/public/components/RBAC/bindings.jsx

```diff
@@ -432,13 +431,12 @@ class BaseEditRoleBindingWithTranslation extends React.Component {
       subjects: [
         {
           apiGroup: 'rbac.authorization.k8s.io',
-          kind: subjectKind || 'User',
-          name: subjectName || '',
+          kind: 'User',
+          name: '',
```

Comment:
I guess I'm unclear on the fix. Why can't be initialize the value to the subject that's currently set in the RoleBinding?

Reply:
@spadgett The values are not available if the context is from Group and User pages. 
This is the PR that set Group and User values - https://github.com/openshift/console/pull/7546 using query parameters.

Reply:
> I guess I'm unclear on the fix. Why can't be initialize the value to the subject that's currently set in the RoleBinding?

The Edit and Duplicate binding makes use of `BindingLoadingWrapper` component and it is connected to `Firehose` and Redux store.  So the subjects is from the store. The Create RoleBinding subjects is using the initialized values  `   
 {
          apiGroup: 'rbac.authorization.k8s.io',
          kind: 'User',
          name: '',
        }`
        
        The exception occurred because Edit and Duplicate doesn't have defined `subjectRef` from  query parameters. The subjects comes from the store and get set in this logic:
        [](https://github.com/openshift/console/blob/master/frontend/public/components/RBAC/bindings.jsx#L423)
        
     To get the Group and User context for Create Role Binding, I introduced `subjectRef` and used it in the subjects initialized in RoleBinding page and handled the null check to prevent Edit and Duplicate from exception. 
 
 I'm thinking that the previous submitted change described above looks clearer. 
 
 Another option might be to rewrite Create Binding component to make use of Redux store and Firehose.
 
 Any thoughts?